style.css

```diff
@@ -86,3 +94,181 @@ li {
 .social-icon {
   margin-right: 10px;
 }
+
+.works-section {
+  padding: 8em 1em;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 5em;
+}
+
+main .project {
+  display: grid;
+  background-color: white;
+  padding: 20px;
+  border-radius: 1em;
+  border: 1px solid #dfe1e6;
+}
+
+.project .project-img {
+  align-items: center;
+}
+
+.project .project-img img {
+  width: 100%;
+}
+
+.project .project-content h3 {
+  text-align: left;
+  font-size: 32px;
+  font-weight: 700;
+  list-style: 44px;
+}
+
+.project .project-content .project-content-list {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+}
+
+.about-skills ul li .active {
+  border: none;
+}
+
+.project .project-content .project-content-list .active {
+  color: #344563;
+}
+
+.about-skills ul li ul {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1em;
+}
+
+.about-skills ul li a {
+  width: 100%;
+  padding: 1em 0;
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.5em;
+  font-weight: 400;
+  color: black;
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.about-skills ul li ul li {
+  width: 100%;
+  padding: 0.5em;
+  background-color: #f7f7f9;
+  display: flex;
+  align-items: center;
+  gap: 1em;
+  border-radius: 0.5em;
+}
+
+.project .project-content .project-content-list li {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 13px;
+  color: #7a869a;
+}
+
+.project .project-content p {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  margin: 20px 0 20px 0;
+}
+
+.project .project-content .project-content-list li a {
+  padding: 10px 10px 10px 0;
+}
+
+.project .project-content .project-language {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.project .project-content .project-language li {
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 10px;
+  font-size: 13px;
+  padding: 0.5rem 1rem;
+  font-weight: bold;
+}
+
+.project .project-content .project-language li a {
+  padding: 20px;
+  text-align: center;
+}
+
+.button {
+  padding: 10px;
+  margin: 20px 0 0 0;
+  border: 1px solid #396df2;
+  color: #396df2;
+  border-radius: 5px;
+}
```

Comment:
- [ ] Please set the background colour for the buttons to white.

Your Design | Figma Design |
:---------------:|:-------------------:|
![a1](https://user-images.githubusercontent.com/39523312/164803709-35e70390-30a6-4d60-b28d-20fa65ac9521.png)| ![a2](https://user-images.githubusercontent.com/39523312/164803711-969958b2-d8e2-4935-a0e0-97b20c7ca4e6.png)

Reply:
solved the issue of the background color